### PR TITLE
Remove logs

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

**Review by vibe13 — APPROVED** [review]
All looks good @janinko ! Thanks for this. Out of curiosity, what is the `buildOutputChecksum` ? Seems to be there since a while, I don't remember what is stands for. Thanks!

File: bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java [hunk 1]
```diff
@@ -338,24 +343,26 @@ private Build getNpmBuild(
 
     private void addLogs(BuildRecord buildRecord, Set<Logfile> logs) {
         String externalBuildID = buildRecord.getId().toString();
-        if (buildRecord.getBuildLogSize() != null && buildRecord.getBuildLogSize() > 0) {
+        long buildLogSize = bifrost.getFinalLogSize(buildRecord.getId().toString(), "build-log");
```
Comment (michalovjan) [inline bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java:346]: the processContext for build logs should be `build-<<buildId>>` right?

File: bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java [hunk 2]
```diff
@@ -338,24 +343,26 @@ private Build getNpmBuild(
...
-                            .size(buildRecord.getBuildLogSize())
-                            .md5(buildRecord.getBuildLogMd5())
+                            .size((int) buildLogSize)
+                            .md5(buildRecord.getbuildOutputChecksum())
                             .build());
         } else {
             logger.warn("Missing build log for BR.id: {}.", externalBuildID);
         }
-        if (buildRecord.getRepourLogSize() != null && buildRecord.getRepourLogSize() > 0) {
+        long alignmentLogSize = bifrost.getFinalLogSize(buildRecord.getId().toString(), "alignment-log");
```
Comment (michalovjan) [inline bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java:358]: the processContext for build logs should be `build-<<buildId>>` right?

File: facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java [hunk 3]
```diff
@@ -629,23 +619,6 @@ public Build getSpecific(String buildId) {
         return build;
     }
 
-    @Override
-    public Page<Build> getAllByStatusAndLogContaining(
```
Comment (michalovjan) [inline facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java:633]: was this used?

File: model/src/main/java/org/jboss/pnc/model/BuildRecord.java [hunk 4]
```diff
@@ -191,18 +191,6 @@ public class BuildRecord implements GenericEntity<Base32LongID> {
     @Column(updatable = false)
     protected Boolean scmBuildConfigRevisionInternal;
 
-    @Lob
-    @Type(type = "org.hibernate.type.TextType")
-    @Basic(fetch = FetchType.LAZY)
-    @LazyGroup("buildLog")
-    private String buildLog;
```
Comment (michalovjan) [inline model/src/main/java/org/jboss/pnc/model/BuildRecord.java:198]: You should add migration scripts to remove the fields from DB.

**Review by michalovjan — CHANGES_REQUESTED** [review]

File: facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java (hunk 3 above)
Comment (janinko) [reply facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java:633]: nope

File: bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java (hunk 1 above)
Comment (janinko) [reply bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java:346]: I belive it's only buildId, right @thescouser89 ?

File: bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java (hunk 2 above)
Comment (janinko) [reply bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java:358]: I belive it's only buildId, right @thescouser89 ?

File: bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java (hunk 1 above)
Comment (thescouser89) [reply bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java:346]: I have a feeling it's just <<buildId>>. But I'm using the IdConverter to switch between process context and the "long id" and it's been working fine.

File: bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java (hunk 2 above)
Comment (thescouser89) [reply bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java:358]: Answered in the previous question.

File: model/src/main/java/org/jboss/pnc/model/BuildRecord.java (hunk 4 above)
Comment (thescouser89) [reply model/src/main/java/org/jboss/pnc/model/BuildRecord.java:198]: Right! But let's keep it after the mgiration for a bit just to be safe.

Comment (janinko) [reply model/src/main/java/org/jboss/pnc/model/BuildRecord.java:198]: added model/src/main/resources/db-update/30000-remove-build-logs.sql

**Review by michalovjan — APPROVED** [review]

**Comment by thescouser89** [comment]
@janinko feel free to merge it whenever you are ready! :)

**Pull request merged**